Sources/Nimble/Matchers/BeCloseTo.swift

```diff
@@ -49,7 +49,13 @@ public class NMBObjCBeCloseToMatcher: NSObject, NMBMatcher {
         })
         let expr = Expression(expression: actualBlock, location: location)
         let matcher = beCloseTo(self._expected, within: self._delta)
-        return try! matcher.matches(expr, failureMessage: failureMessage)
+
+        do {
+            return try matcher.matches(expr, failureMessage: failureMessage)
+        } catch let error {
+            failureMessage.stringValue = "unexpected error thrown: <\(error)>"
```

Comment:
This error text is repeated, is there something we can extract here?

Reply:
It makes sense 👍(will be done in a separate PR)